docs/reference-Session.html

```diff
@@ -174,7 +174,7 @@ <h1 class="title">Session</h1>
 in this <tt class="docutils literal">add_torrent_params</tt> object will take precedence over
 whatever states are saved in the resume data. For instance, the
 <tt class="docutils literal">paused</tt>, <tt class="docutils literal">auto_managed</tt>, <tt class="docutils literal">sequential_download</tt>, <tt class="docutils literal">seed_mode</tt>,
-<tt class="docutils literal">super_seeding</tt>, <tt class="docutils literal">max_uploads</tt>, <tt class="docutils literal">max_connections</tt>,
```

Comment:
oh, I guess this is the resume data documentation. that's in one of the .rst files